### ISSUE TITLE
rotating HTerrainDetailLayer objects to camera

### ISSUE THREAD

**Issue by iliakravets677** [Zylann/godot_heightmap_plugin#350]
Good afternoon, I ask you to add a similar function to your plugin, because I need it, but I can’t implement it in any way, I have to write to you. To make objects like Terrain Detail rotate towards the camera. please add this function to your plugin. an example of how it should look on the screenshot, grass and other objects should rotate towards the player
![изображение_2022-11-12_120544151](https://user-images.githubusercontent.com/56079131/201469180-2d327c3e-574d-4bf6-be47-b15de3033459.png)

**Comment by Zylann** [comment]
They aren't rotated on the screenshot, or at least not in a way that makes it obvious, but I think I see what you mean. You want grass to be "billboards", to always face the camera. To do this a new variant of the shader is required.

**Comment by iliakravets677** [comment]
> They aren't rotated on the screenshot, or at least not in a way that makes it obvious, but I think I see what you mean. You want grass to be "billboards", to always face the camera. To do this a new variant of the shader is required.

yes that's it, i tried to add a shader, but the billboard mod still didn't work
